controllers/jetstream/consumer.go

```diff
@@ -133,56 +131,51 @@ func (c *Controller) processConsumerObject(cns *apis.Consumer, jsmc jsmClient) (
 		servers := spec.Servers
 		if c.opts.CRDConnect {
 			// Create a new client
-			opts := make([]nats.Option, 0)
-			opts = append(opts, nats.Name(fmt.Sprintf("%s-con-%s-%d", c.opts.NATSClientName, spec.DurableName, cns.Generation)))
+			natsCtx := &natsContext{}
+			natsCtx.Name = fmt.Sprintf("%s-con-%s-%d", c.opts.NATSClientName, spec.DurableName, cns.Generation)
```

Comment:
This is going to kill pooling - name is part of the pooling hash

Reply:
Fixed here: https://github.com/nats-io/nack/pull/152/commits/0768570c8a0935b8cf0f76d1c776fac30c48e466